docs/typescript/clients.md

```diff
@@ -207,7 +210,7 @@ const handle = await client.start(scheduledWorkflow, {
 
 :::info Should I use Cron Workflows or Timers?
 
-This section is specifically about [Temporal Cron Jobs](/docs/content/what-is-a-temporal-cron-job/), Workflows that have the `cronSchedule` option set in Temporal.
+This section is specifically about <preview page={WhatIsATemporalCronJob}>Temporal Cron Jobs</preview>, Workflows that have the `cronSchedule` option set in Temporal.
 Since Temporal Workflows have [Timers](/docs/typescript/workflows#timers), can loop indefinitely, and can spawn [Child Workflows](/docs/typescript/workflows#child-workflows), it is natural to ask when to use which.
```

Comment:
```suggestion
Because Temporal Workflows have [Timers](/docs/typescript/workflows#timers), can loop indefinitely, and can spawn [Child Workflows](/docs/typescript/workflows#child-workflows), it is natural to ask when to use which.
```